Members/anshdhinhgra47.md

```diff
@@ -0,0 +1,28 @@
+## About
+Hello Everyone! I am an incoming freshman at the department of Computer Science, The University Of Waterloo, Canada. I have always been interested in Web Development and Artificial Intelligence. As of now, I currently working on an Anti-Depression Chatbot.
+
+
+## Place
+New Delhi, India
+
+## Skills
+-HTML, CSS
```

Comment:
Here in the bullet points too. it should be `- HTML, CSS` instead of `-HTML, CSS`

Reply:
For all the bullet points ?